third_party/move/move-compiler-v2/tests/checking/variants/variants_test_parse_err2.move

```diff
@@ -0,0 +1,12 @@
+module 0x815::m {
+
+    enum Color {
+        RGB{red: u64, green: u64, blue: u64},
+        Red,
+        Blue,
+    }
+
+    fun test_red_or_rgb(c: Color): bool {
+        c is Red
```

Comment:
Why does this fail?  Do we have a grammar conflict if we allow it?

Reply:
Yeah, very weird Move syntax, it always has to be `(x as T)` or `(x: T)` and `(x is T)` follows this convention. We may want to look at relaxing this in general). I think the grammar can be made unique without this.